### PR TITLE
Hedera <> WalletConnect

### PR DESCRIPTION
**Pull request by hgraphql**
# Reference implementation
This PR intends to serve as the reference implementation of https://specs.walletconnect.com/2.0/blockchain-rpc/hedera-rpc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,34 +1,69 @@
...
   },
+  "types": "./dist/types/index.d.ts",
+  "author": "Hgraph <support@hgraph.io>",
   "keywords": [
     "hedera",
     "walletconnect",
     "hashgraph"
   ],
-  "license": "Apache-2.0",
+  "license": "MIT",
```
Comment (nathanklick) [inline package.json:22]: The Hedera standard license for all OSS projects is `Apache 2.0`. Please revert this back to an `Apache 2.0` license.

File: LICENSE
Comment (nathanklick) [inline LICENSE:1]: Please revert this change. The Hedera standard license for all OSS projects in an `Apache 2.0` license.

**Review by nathanklick — CHANGES_REQUESTED** [review]

File: .github/workflows/prettier.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: prettier
```
Comment (nathanklick) [inline .github/workflows/prettier.yml:1]:
```suggestion
name: Prettier
```

File: .github/workflows/prettier.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  prettier-check:
+    runs-on: ubuntu-latest
```
Comment (nathanklick) [inline .github/workflows/prettier.yml:13]:
```suggestion
  formatting-check:
    name: Formatting Check
    runs-on: [self-hosted, Linux, medium, ephemeral]
```

File: .github/workflows/prettier.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+name: prettier
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
```
Comment (nathanklick) [inline .github/workflows/prettier.yml:9]:
```suggestion
on:
  push:
  pull_request:
  
permissions:
  contents: read
```

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -7,6 +7,8 @@ jobs:
   test-node:
     name: Unit Tests
     runs-on: [self-hosted, Linux, medium, ephemeral]
+    permissions: write-all
```
Comment (nathanklick) [inline .github/workflows/test.yml:10]:
```suggestion
    permissions: 
       checks: write
       pull-requests: write
       contents: read
```

**Review by nathanklick — CHANGES_REQUESTED** [review]

**Comment by hgraphql** [comment]
thanks @nathanklick,

all code changes have been applied

File: .github/dependabot.yml
[the remark this answers is not in the document]
Comment (hgraphql) [reply .github/dependabot.yml:1]: great, thanks @nathanklick 

I've added the file and section.

**Review by nathanklick — APPROVED** [review]
@hgraphql LGTM now, thanks for making all the changes!

**Comment by hgraphql** [comment]
> @hgraphql LGTM now, thanks for making all the changes!

@nathanklick awesome, thanks for the help!

**Comment by nathanklick** [comment]
@gregscullard @Ashe-Oro All my concerns are resolved. Please review and merge when you are ready.

**Pull request merged**